docs/reference/action_list.rst

```diff
@@ -143,88 +143,21 @@ Options
-|           |   name              | Route name                                                            |
-+           +                     +                                                                       +
-|           |   parameters        | Route parameters                                                      |
-+           +---------------------+-----------------------------------------------------------------------+
-|           | hide_protocol       | Hide http:// or https:// (default: false)                             |
-+-----------+---------------------+-----------------------------------------------------------------------+
-
-If you have the SonataDoctrineORMAdminBundle installed, you have access
-to more field types, see `SonataDoctrineORMAdminBundle Documentation`_.
-
-.. note::
-
-    It is better to prefer non negative notions when possible for boolean
-    values so use the ``inverse`` option if you really cannot find a good enough
-    antonym for the name you have.
++--------------------------------------+---------------------+-----------------------------------------------------------------------+
+| Type                                 | Options             | Description                                                           |
++======================================+=====================+=======================================================================+
+| ``ListMapper::TYPE_ACTIONS``         | actions             | List of available actions                                             |
++                                      +                     +                                                                       +
+|                                      |   edit              | Name of the action (``show``, ``edit``, ``history``, ``delete``, etc) |
++                                      +                     +                                                                       +
+|                                      |     link_parameters | Route parameters                                                      |
++--------------------------------------+---------------------+-----------------------------------------------------------------------+
+| ``ListMapper::TYPE_BATCH``           |                     | Renders a checkbox                                                    |
++--------------------------------------+---------------------+-----------------------------------------------------------------------+
+| ``ListMapper::TYPE_SELECT``          |                     | Renders a select box                                                  |
++--------------------------------------+---------------------+-----------------------------------------------------------------------+
+| ``TemplateRegistry::TYPE_*``         |                     | See Field types chapter                                               |
```

Comment:
How can I add a link here ?

Reply:
If the chapter is a single rst file, you could use `:doc:`, otherwise if it's a part of a rst file, you would use `:ref:`. More on this at https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#ref-role